plugins/modules/network/routeros/routeros_facts.py

```diff
@@ -68,6 +68,19 @@
   description: The configured hostname of the device
   returned: always
   type: str
+ansible_net_arch:
+  description: The CPU architecture of the device
+  returned: always
+  type: str
+  version_added: 1.1.0
+ansible_net_uptime:
+  description: The uptime of the device
+  returned: always
+  type: str
```

Comment:
This needs a version_added, and so do all other new ones.

Reply:
Do I understand correctly that when adding new variables, you need to specify which version they appeared in? In this case, the question is, where are minor and major version changes tracked? In what case do they change? You indicated above ( version_added: 1.1.0) where you got this information? Do I need to specify it in the Routing function?

Reply:
The next release of community.network will be 1.1.0, so you need to document these return values as added for community.network 1.1.0. You can find information on versioning here: #81

(This is about the collection version, not about for which librouteros / RouterOS versions which actually support them.)

Reply:
BTW, 1.1.0 has been released, so you need to add `version_added: 1.2.0` for all new return values.